cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala

```diff
@@ -211,7 +211,8 @@ class QueryPlanTest extends DocumentingTestBase with SoftReset {
     profileQuery(
       title = "Distinct",
       text =
-        """The `Distinct` operator removes duplicate rows from the incoming stream of rows.""".stripMargin,
+        """The `Distinct` operator removes duplicate rows from the incoming stream of rows.
+          |To ensure only distinct elements are returned, `Distinct` needs to eagerly pull in all data from its source and build up state, which will lead to increased memory pressure in the system.""".stripMargin,
```

Comment:
I've double checked the code again and realised that what I suggested this to say is not true. DISTINCT does not eagerly load anything. It does build up the same state, but lazily. So, it's still not great from a memory consumption POV, but less awful than eager loading at least.